build_with_cmake/napi/hello.cc

```diff
@@ -0,0 +1,23 @@
+#include <assert.h>
+#include <node_api.h>
+
+napi_value Method(napi_env env, napi_callback_info info) {
```

Comment:
This method should be static.

Reply:
Fixed in https://github.com/nodejs/node-addon-examples/pull/114.